playbooks/ORCHESTRATION.md

```diff
@@ -0,0 +1,219 @@
+```
+$REPO
+├── keys
+├── repository
+└── staged
+    ├── root.json
+    ├── targets
+    │   └── $TARGET
+    ├── targets.json
+    ├── staging.json
+    ├── rekor.json   
+    └── revocation.json 
+```
+
+Manually check for:
+* The expected root and targets expirations.
+* The expected root and targets versions.
+* The expected keyholders and placeholder signatures.
+* The expected target and delegation files. Check the termination, paths, and targets on each delegation.
+
+<!-- TODO: Add playbook for disaster/recovery steps. -->
+
+### Hardware Key Signing
+
+Next, the root and targets file must be signed. Ask each root keyholder to run:
+
+```bash
+./scripts/step-0.sh && ./scripts/step-2.sh
+```
+
+This will create the following structure.
+```
+${REPO}/keys
+└── 89957089
+    ├── 89957089_device_cert.pem
+    ├── 89957089_key_cert.pem
+    └── 89957089_pubkey.pem
+```
+
+Verify their PRs with the PR number as the argument to the script:
+
+```bash
+./scripts/verify.sh $PR
+```
+
+You should expect 1 signature added to root and targets on each PR.
+
+After each of the root keyholder PRs are merged, run verification at main:
+
+```bash
+./scripts/verify.sh
+```
+
+and verify that the root and targets are fully signed.
+
+## Step 3: Delegations
+
+After root and targets signing, the delegation files must be signed.
+
+```bash
+./scripts/step-3.sh
+```
+
+This will create a PR signing the delegations. Verify the PR with the PR number as the argument to the script:
+
+```bash
+./scripts/verify.sh $PR
+```
+
+and check that the delegation was successfully signed.
+
+
+## Step 4: Snapshotting and Timestamping
+
+Next, the metadata will need to be snapshotted and timestamped. Run 
+
+```bash
+./scripts/step-4.sh
+```
+
+This will create a PR signing the snapshot and timestamp files. Verify the expirations and the signatures:
+
+```bash
+./scripts/verify.sh $PR
+```
+
+## Step 5: Publishing
+
+This final step will commit the TUF repository metadata and move it to the top-level folder `repository/repository/`.
+
+```bash
+./scripts/step-5.sh
+```
+
+This will create a PR moving the files. Verify that the TUF client can update to the new metadata with:
+
+```bash
+./scripts/verify.sh $PR
+```
+
+## Post-ceremony Steps
+
+1. If any root keyholders have changed, update the [current root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders) with their name, key ID, and location of their key material.
+
+2. If any targets have changed, update them and their usage in the table containing the [repository structure](https://github.com/sigstore/root-signing#tuf-repository-structure).
+
+3. Announce the root rotation on twitter and the community meeting, and thank the keyholders!
+
+4. Schedule the next root signing event 3 months from now on the calendar. The root expires in  months. Schedule a testing event for the week before.
```

Comment:
3 months? Or 5? Also missing the root expiration

Reply:
Ah! I'll specify one month before expiration (I forgot expiration is still at 6 months)